initial/contract-rest-service/src/main/java/hello/PersonService.java

```diff
@@ -0,0 +1,25 @@
+package hello;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.stereotype.Service;
+
+@Service
+class PersonService {
+
+	private Map<Long, Person> personMap = new HashMap<>();
+	
+	@PostConstruct
```

Comment:
Let's move this to constructor? We could also make the map final

Reply:
fixed. thanks